test/integration/ext_deletion_test.go

```diff
@@ -46,79 +42,61 @@ func TestExtDeletion(t *testing.T) {
 	if _, err := testserver.CreateNewProject(clusterAdminClient, *clusterAdminClientConfig, projName, "admin"); err != nil {
 		t.Fatalf("unexpected error creating the project: %v", err)
 	}
+	_, projectAdminKubeClient, _, err := testutil.GetClientForUser(*clusterAdminClientConfig, "admin")
```

Comment:
I wanted to do the creations as the project admin, to make sure they are able to